src/app/projects-list/page.tsx

```diff
@@ -0,0 +1,117 @@
+'use client';
+import { Button } from '@/components/button/button';
+import { Card } from '@/components/card/card';
+import { classnames } from '@/utils/classnames';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+interface ProjectCategory {
+  projectId: number;
+  categoryId: number;
+  categoryName: string;
+}
+
+interface User {
+  userId: number;
+  username: string;
+  passwordHash: string;
+  email: string;
+  fullName: string;
+  bio: string | null;
+  userType: string;
+  createdAt: string;
+}
+
+interface Project {
+  projectId: number;
+  userId: number;
+  user: User;
+  title: string;
+  description: string;
+  price: number;
+  createdAt: string;
+  projectCategories: ProjectCategory[];
+}
+
+const Projects = () => {
+  const [projects, setProjects] = useState<Project[]>([]);
+
+  useEffect(() => {
+    const fetchProjects = async () => {
```

Comment:
please use tanstackquery useQuery hook to call apis.  
It should have already been used in the project so you can search for it.